packages/core/upload/admin/src/pages/App/MediaLibrary.js

```diff
@@ -255,11 +300,38 @@ export const MediaLibrary = () => {
             />
           )}
 
-          {canRead && (
+          {/* TODO: fix AssetListTable should handle no assets views (loading) */}
+          {canRead && !isGridView && (assetCount > 0 || folderCount > 0) && (
+            <TableList
+              assetCount={assetCount}
+              folderCount={folderCount}
+              indeterminate={indeterminateBulkSelect}
+              onChangeSort={handleChangeSort}
+              onChangeFolder={(folderID) => push(getFolderURL(pathname, query, folderID))}
+              onEditAsset={setAssetToEdit}
+              onEditFolder={handleEditFolder}
+              onSelectOne={selectOne}
+              onSelectAll={handleBulkSelect}
+              rows={
+                // TODO: remove when fixed on DS side
+                // when number of rows in Table changes, the keyboard tab from a row to another
+                // is not working for 1st and last column
+                !assetsLoading && !foldersLoading ? [...folders, ...assets] : []
```

Comment:
Do we have an issue for that in the DS?

Reply:
my bad I thought I removed that already - fixed by Josh in the DS!

Reply:
[fixed](https://github.com/strapi/strapi/pull/15085/commits/dc055b43710f3606b5f450f318f99638d50ae00e)